lib/xeroizer/models/organisation.rb

```diff
@@ -29,6 +29,8 @@ class Organisation < Base
       string    :registration_number
       string    :timezone
       datetime  :created_date_utc, :api_name => 'CreatedDateUTC'
+      string    :sales_tax_basis
+      string    :sales_tax_period
```

Comment:
Probably. Would prefer to receive a PR for it though than have a bunch of
issues created.

On Monday, March 23, 2015, CloCkWeRX <notifications@github.com> wrote:

> In lib/xeroizer/models/organisation.rb
> <https://github.com/waynerobinson/xeroizer/pull/184#discussion_r26911910>:
>
> > @@ -29,6 +29,8 @@ class Organisation < Base
> >        string    :registration_number
> >        string    :timezone
> >        datetime  :created_date_utc, :api_name => 'CreatedDateUTC'
> > +      string    :sales_tax_basis
> > +      string    :sales_tax_period
>
> Ditto http://developer.xero.com/documentation/api/types/#SalesTaxPeriod
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/waynerobinson/xeroizer/pull/184/files#r26911910>.
>